test/framework/src/main/java/org/elasticsearch/test/transport/CapturingTransport.java

```diff
@@ -253,10 +253,6 @@ protected void onSendRequest(long requestId, String action, TransportRequest req
         capturedRequests.add(new CapturedRequest(node, requestId, action, request));
     }
 
-    protected boolean nodeConnected(DiscoveryNode discoveryNode) {
```

Comment:
This is a port of some changes from the `zen2` branch for easier future merging. Please do not back this out.